Main.css

```diff
@@ -0,0 +1,3 @@
+.Header {
+  color: gold;
+}
```

Comment:
[OPTIONAL] It is considered bad practice to use color names. I suggest you change this color to its `rgb` or `hex` equivalent. Also, I recommend you lowercase your classes, it's the industry standard.